diff --git a/dlls/mshtml/nsembed.c b/dlls/mshtml/nsembed.c
index 651229bba74..4d4277d88d4 100644
--- a/dlls/mshtml/nsembed.c
+++ b/dlls/mshtml/nsembed.c
@@ -56,7 +56,7 @@ WINE_DECLARE_DEBUG_CHANNEL(gecko);
 
 #ifdef __i386__
 #define GECKO_ARCH_STRING "x86"
-#elif defined(__x86_64__)
+#elif defined(__x86_64__) || defined(__aarch64__)
 #define GECKO_ARCH_STRING "x86_64"
 #else
 #define GECKO_ARCH_STRING ""
